Plugins/FpeMonitoring/CMakeLists.txt

```diff
@@ -28,6 +30,67 @@ else()
     if(Backtrace_FOUND)
 
       # check if we need to link against bracktrace or not
+      set(backtrace_include "")
+
+
+      file(WRITE
+        "${CMAKE_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/CMakeTmp/backtrace.cpp"
+        "
+        #include <boost/stacktrace.hpp>
+        int main() {}
+        " )
+
+
+      message(CHECK_START "Does backtrace work with the default include")
+
+      try_compile(_backtrace_default_header "${CMAKE_BINARY_DIR}"
+          "${CMAKE_BINARY_DIR}${CMAKE_FILES_DIRECTORY}/CMakeTmp/backtrace.cpp"
+          LINK_LIBRARIES ${dl_LIBRARY}
+          COMPILE_DEFINITIONS -DBOOST_STACKTRACE_USE_BACKTRACE
+          OUTPUT_VARIABLE __OUTPUT)
+
+
+      if(_backtrace_default_header)
+        message(CHECK_PASS "yes")
+      else()
+        message(CHECK_FAIL "no")
+
+        file(GLOB hints "/usr/lib/gcc/*/*/include")
```

Comment:
Not sure if I understand this code correctely: Do we search here for the file, and if found, we use a hardcoded path?
Why not something like `if(EXISTS "/usr/lib/gcc/x86_64-linux-gnu/11/include/backtrace.h")`?

Reply:
Not quite. The `find_file` will look in the standard paths. I'm using this glob to supplement the search paths in `find_file` with all folder matching `/usr/lib/gcc/*/*/include`, because that's where @andiwand and I found this header locally. 

Does that make sense?

Reply:
Hmm not 100% yet, because I would excpect then
```cmake
set(backtrace_include "-DBOOST_STACKTRACE_BACKTRACE_INCLUDE_FILE=${backtrace_header})
```
and not the hardcoded path...

Reply:
Ah right, I forgot to change that. Changed now.